client/pages/project.jsx

```diff
@@ -4,19 +4,86 @@ export default class Project extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      project: null
+      project: null,
+      details: []
     };
   }
 
   componentDidMount() {
     fetch(`/api/home/${this.props.projectId}`)
       .then(res => res.json())
       .then(result => this.setState({ project: result }));
+
+    fetch(`/api/projects//${this.props.projectId}`)
+      .then(res => res.json())
+      .then(result => this.setState({ details: this.state.details.concat(result) }));
   }
 
   render() {
+
     return (
-      <div>test</div>
+      <div>
+        <nav className="navbar navbar-expand-xl navbar-color navbar-height">
+          <i className="fas fa-bars white fa-2x"></i>
+        </nav>
```

Comment:
You can leave this out until it does something. Including extra code gives your reviewer more work. If you had to understand code you were given, would you prefer more code or less code?